Documentation/ApiOverview/RequestLifeCycle/Typo3Request.rst

```diff
@@ -1,73 +1,241 @@
-.. include:: /Includes.rst.txt
-.. index::
-   Request handling; request object
-   $GLOBALS; TYPO3_REQUEST
-.. _typo3-request:
+..  include:: /Includes.rst.txt
+..  index::
+    Request handling; request object
+    $GLOBALS; TYPO3_REQUEST
+..  _typo3-request:
 
 ====================
 TYPO3 request object
 ====================
 
-The TYPO3 request object is a PSR-7 based `ServerRequest` object containing a TYPO3-specific attribute object for normalized
-server parameters. These normalized parameter for instance resolve variables
-if the instance is behind a reverse proxy.
+The TYPO3 request object is an implementation of the PSR-7 based
+`\Psr\Http\Message\ServerRequestInterface` containing TYPO3-specific attributes.
 
-.. important::
+The request object is passed to controllers. The attributes can be retrieved via
```

Comment:
What kind of controllers do we talk about here? Extbase Controllers?

Reply:
It will be reworked in one of the next iterations. For now I just copied the existing text. First, I want to add the frontend attributes. Next it will be refined.